### ISSUE TITLE
Arrays should have a values object

### ISSUE THREAD

**Issue by jamesrkg** [Quantipy/quantipy#277]
When ``read_dimensions`` or ``read_decipher`` converts a ``grid`` to a quantipy ``array``, the resulting ``array mask`` should have its own values object.

```python
{
    "items": [
        {
            "properties": {}, 
            "source": "columns@qa6[{qa6_1}].qa6_grid", 
            "text": {
                "en-GB": "I like cats."
            }
        }, 
        {
            "properties": {}, 
            "source": "columns@qa6[{qa6_2}].qa6_grid", 
            "text": {
                "en-GB": "I like dogs."
            }
        }, 
        {
            "properties": {}, 
            "source": "columns@qa6[{qa6_3}].qa6_grid", 
            "text": {
                "en-GB": "I like birds."
            }
        }, 
        {
            "properties": {}, 
            "source": "columns@qa6[{qa6_5}].qa6_grid", 
            "text": {
                "en-GB": "I like fish."
            }
        }
    ], 
    "properties": {
        "roworder": "randomize"
    }, 
    "text": {
        "en-GB": "Please indicate the extent to which you agree or disagree"
    }, 
    "type": "array", 
    "values": "lib@values@qa6.qa6_grid"
}
```

**Comment by jamesrkg** [comment]
This will be resolved by the PR that merges **i203-i206-grid_summar_intro**.

**Comment by jamesrkg** [comment]
Resolved by #278.
